src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -24,20 +27,18 @@ static Https_ClientCredentialTypeTests()
 #if FULLXUNIT_NOTSUPPORTED
     [Fact]
 #else
-    [ConditionalFact(nameof(Root_Certificate_Installed), nameof(Basic_Authentication_Available))]
+    [ConditionalFact(nameof(Root_Certificate_Installed))]
```

Comment:
Why did you remove this ConditionalFact?  Do we know Basic_Authentication_Available is true for all platforms, both self-host and IIS-host?